src/Store/Store.jsx

```diff
@@ -0,0 +1,14 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import missionReducer from './reducers';
+import rocketReducer from './rocketReducer';
+
+const store = configureStore({
+  reducer: {
+    mission: missionReducer,
+    // dragon: dragonreducer,
+    rockets: rocketReducer,
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.